build.sh

```diff
@@ -6,7 +6,6 @@ fi
 
 ## Build infrastructure ##
 docker-compose up -d
-sleep 10
```

Comment:
I think you still need the sleep, since it takes time to settle. Healthcheck + sleep?

Reply:
I don't think so, each container runs only if previous is "healthy", in this case it means that REST API is accessible. When we creating cluster we only care about master-node to be "healthy" cause we're making a REST API call to it